src/main/scala/ai/privado/utility/UnresolvedReportUtility.scala

```diff
@@ -33,20 +33,20 @@ import io.shiftleft.codepropertygraph.generated.Cpg
 import ai.privado.utility.Utilities.getFileNameForNode
 
 object UnresolvedReportUtility {
-  def reportUnresolvedMethods(xtocpg: Try[Cpg], filename: String): Unit = {
+  def reportUnresolvedMethods(xtocpg: Try[Cpg], statfilepath: String): Unit = {
     var total                    = 0
     var unresolvedSignatures     = 0
     var unresolvedNamespaces     = 0
     var unresolvedSignaturesList = ListBuffer[String]()
     var unresolvedNamespacesList = ListBuffer[String]()
 
-    val java_unresolved_signature = "(?i)(.*)(unresolved)(signature)(.*)"
-    val java_unresolved_namespace = "(?i)(.*)(unresolved)(namespace)(.*)"
-    val python_unresolved         = "(?i)(.*)(unknownfullname)(.*)"
+    val unresolved_signature = "(?i)(.*)(unresolved)(signature)(.*)"
```

Comment:
Can we follow the Java variable naming convention for variables `unresolvedSignature` like this